website/docs/r/nocode_module.html.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_nocode_module"
+description: |-
+  Manages no code for registry modules
+---
+
+# tfe_nocode_module
+
+Creates, updates and destroys no code for registry modules.
```

Comment:
Does this resource `create a no code for registry modules`? or `creates a no-code registry module`, are the two objects different or referring to the same thing?

If referring to 2 objects, then I'd suggest: `Creates, updates and destroys no-code module for registry modules.`, WDYT?